backend/curriculum_tracking/management/commands/import_curriculum.py

```diff
@@ -0,0 +1,78 @@
+from core.models import Curriculum
+from curriculum_tracking.models import (
+    CurriculumContentRequirement,
+    ContentItemOrder,
+    ContentItem,
+)
+from curriculum_tracking.card_generation_helpers import get_ordered_content_items
+import json
+from django.core.management.base import BaseCommand
+import pandas as pd
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        json_file = "dev_helpers/data/intro-to-tilde-course.json"
+        save_curriculum_to_db(json_file)
+
+
+def save_curriculum_to_db(json_file):
+    with open(json_file) as f:
+        data = json.load(f)
+
+    # keys in dictionary `data` (4) `content_item_orders`, `content_items`, `curriculum`, `curriculum_content_requirements`
+    create_content_items(data['content_items'])
+    create_content_item_orders(data['content_item_orders'])
+    create_curriculum(data['curriculum'])
+    create_curriculum_content_requirements(data['curriculum_content_requirements'])
+
+def find_url_index(link, model):
+    if link == None:
+        return 
+    else:
+        for index, value in enumerate(model.objects.all()):
+            if value.url == link:
+                return model.objects.filter(url=link).first()
+
+def create_content_items(data):
+    for content in data:
+        content_items, created = ContentItem.objects.get_or_create(
+            # available_flavours = ','.join(content['available_flavours']),
```

Comment:
are there no available flavours in the export?

Reply:
there are no flavours in the provided json file, they are all empty

Reply:
That's true. But the import script should be the opposite of this script:

curriculum_tracking/management/commands/export_curriculum.py

If flavours are added it should still work. We might want to use this to export and import other courses too. I suppose for now it's ok because we dont *need* the flavours right now. But there is a good chance we will have to revisit this work.

Reply:
Can you please make an issue that details all the fields that were skipped over.

I think it should just talk about flavours and tags. 

Once that issue exists let m know then I'll merge this